locales/en-GB.ts

```diff
@@ -77,9 +77,9 @@ export default defineI18nLocale(() => {
       clearAllCategories: 'Hide all categories',
       back: 'Back',
       backToMenuFavorites: 'Show menu',
-      backToMenuExplorer: 'Search by Categories',
-      backToMenuFavoritesMobile: 'Exit Favorites',
-      backToMenuExplorerMobile: 'Search by Categories',
+      backToMenuExplorer: 'Search by categories',
+      backToMenuFavoritesMobile: 'Exit favorites',
+      backToMenuExplorerMobile: 'Exit \'Nearby\'',
```

Comment:
```suggestion
      backToMenuExplorerMobile: 'Exit "Nearby"',
```

Reply:
why not using « » ? it's language specific ?

Reply:
No, only french typo do that.

Reply:
Yes it is by language.